analysis_driver/dataset.py

```diff
@@ -458,6 +459,14 @@ def run_elements(self):
             )
         return self._run_elements
```

Comment:
With the new aggregation, we should be able to remove self.run_elements and only use self.sample